life_server/Functions/Gangs/fn_removeGang.sqf

```diff
@@ -6,7 +6,7 @@
     Description:
     Removes gang from database
 */
-private ["_group","_groupID"];
+private ["_group","_groupID","_result"];
 _group = param [0,grpNull,[grpNull]];
```

Comment:
this can become params [
["_group",grpNull,[grpNull]]
];

Reply:
Are you sure that even for only 1 is better params insted of param?

Reply:
For consistency alone, yes.

Reply:
And performance yes.